ui/client/components/modals/GenericModalDialog.tsx

```diff
@@ -71,12 +71,14 @@ class GenericModalDialog extends React.Component<Props, State> {
       >
         <div className="draggable-container">
           <Draggable bounds="parent" handle=".modal-draggable-handle">
-            <div className={`espModal ${this.props.style || "confirmationModal"}`}>
-              {this.props.header ? (
-                <div className="modal-title modal-draggable-handle" style={{color: "white", backgroundColor: "#70C6CE"}}>
-                  <span>{this.props.header}</span>
-                </div>
-              ) : null}
+            <div className={`espModal ${this.props.style || "confirmationModal"}`} data-testid="modal">
```

Comment:
Can we also refactor `espModal` to `nussknacker-modal` and the same with `confirmation-modal` to `confirmation-modal`? :)

Reply:
I will not touch the styles topic now. next pr will bring visual changes